.github/workflows/publish-corfu.yml

```diff
@@ -42,7 +42,13 @@ jobs:
         continue-on-error: true
 
       - name: Prepare Corfu
-        run: ./mvnw clean install -Pdocker -DskipTests -Dmaven.javadoc.skip=true -Dcheckstyle.skip -T 1C
+        run: ./mvnw clean install -Pdocker -pl \!samples -Dmaven.javadoc.skip=true -Dcheckstyle.skip -T 1C
+
```

Comment:
Considering the flaky tests, I think this will make the artifacts hosting job unstable.

Reply:
I thought the entire job will fail and we can rerun it when it fails. Since it is including only the unit tests, I thought it is okay to rerun them.

Reply:
Yeah I agree.. but hmm I thought as there is no gating on the jobs merged on master branch, we might forget it sometimes...and for code coverage report I thought it's okay to re-trigger when needed (given that it failed sometimes in between)